wherehows-backend/app/controllers/Application.java

```diff
@@ -76,4 +81,37 @@ public static Result index() {
   public static Result healthcheck() {
     return ok("GOOD");
   }
+
+  public static Result deploylibsbackend() {
+    String env = System.getenv("WHZ_APP_HOME");
+    String libpath = env + "/lib";
```

Comment:
One suggestion is to use the camel case as Java standard. E.g. libPath, commitFile